utils/compare_results.py

```diff
@@ -133,18 +134,34 @@ def get_results(xml: ElementTree.ElementTree) -> dict:
     return rules
 
 
-def get_identifiers(xml: ElementTree.ElementTree) -> dict:
+def get_identifiers(rule: ElementTree.ElementTree) -> dict:
+    try:
+        return rule.find('xccdf-1.2:ident', PREFIX_TO_NS).text
+    except AttributeError:
+        return "MISSING_IDREF"
+
+
+def get_check_ids(rule: ElementTree.ElementTree) -> dict:
```

Comment:
I feel this will return either dict or str

Reply:
Nice catch, it returns a string